client/me/purchases/manage-purchase/index.jsx

```diff
@@ -288,7 +288,7 @@ const ManagePurchase = React.createClass( {
 			}
 
 			return (
-				<span className="manage-purchase__payment-info">
+				<span className="manage-purchase__detail manage-purchase__payment-info">
```

Comment:
I think this should only have one class, `manage-purchase__detail`. We can style the child by using the `PaymentLogo` classname.